jest.config.js

```diff
@@ -12,13 +12,17 @@ module.exports = {
   coverageDirectory: './artifacts/coverage-jest/',
   coverageReporters: ['lcov'],
   reporters: ['jest-junit', 'default'],
-  transform: { '^.+\\.(js|jsx)$': path.join(__dirname, './test/jest/jest-transformer.js') },
-  transformIgnorePatterns: [`/node_modules/(?!${esModules}|ky)`],
+  transform: {
+    '^.+\\.(js|jsx|ts|tsx)$': path.join(__dirname, './test/jest/jest-transformer.js'),
```

Comment:
For TS we need to also add `@babel/preset-typescript` preset to jest's `babel.config.js`